rs/bitcoin/ckbtc/minter/src/updates/retrieve_btc.rs

```diff
@@ -174,8 +174,13 @@ pub async fn retrieve_btc(args: RetrieveBtcArgs) -> Result<RetrieveBtcOk, Retrie
     }
 
     let _guard = retrieve_btc_guard(caller)?;
-    let (min_retrieve_amount, btc_network, kyt_fee) =
-        read_state(|s| (s.retrieve_btc_min_amount, s.btc_network, s.kyt_fee));
+    let (min_retrieve_amount, btc_network, kyt_fee) = read_state(|s| {
+        (
+            s.fee_based_retrieve_btc_min_amount,
+            s.btc_network,
+            s.kyt_fee,
+        )
+    });
 
     let min_amount = max(min_retrieve_amount, kyt_fee);
```

Comment:
But is there any previous check before this line?
If there is none, then it makes sense to check here that the amount in the request is not lower than the KYT fee.
It is a bit of a hypothetical situation because the minimum retrieval amount will always be greater than the KYT fee.

Reply:
I suggest we leave the `max` as it is just to be sure.

Reply:
sounds good, I was asking for my own understanding.